### DISCUSSION TITLE
Import only the charts needed

### DISCUSSION THREAD

**Discussion by Snugug** [ChartsCSS/charts.css#64] [Q&A]
Similar to tree shaking for JS libraries, is it possible to only include the specific pieces need to make, say, a line chart work? I presume so if the `src` was distributed via the `npm` package, but it's not clear to me if only importing the `general`, partials, the `component` partials that I want to use, and the `chart` partial I need would actually work.

**Comment by russelporosky** [comment]
You'd need a build process to run a CSS tree-shaker (I think Webpack's `style-loader` can do this).
